config/jobs/kubernetes-sigs/downloadkubernetes/downloadkubernetes-presubmits.yaml

```
@@ -0,0 +1,18 @@
+presubmits:
+  kubernetes-sigs/downloadkubernetes:
+  - name: pull-downloadkubernetes-verify-index
+    always_run: true
+    optional: false
+    decorate: true
+    path_alias: "sigs.k8s.io/downloadkubernetes"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20201208-8b354d9-master
+        command:
+          - "runner.sh"
+          - "./scripts/check-index.sh"
```

Same here about comparing presubmits in the other RelEng repos for guidance.

Prefer:
```suggestion
          - make
          - verify-index
```

and adding a make target for this in the repo.

---

done
then added the makefile rule here: https://github.com/kubernetes-sigs/downloadkubernetes/pull/22

and then added a second job to run the verifies